web/src/routes/System.jsx

```diff
@@ -143,13 +143,15 @@ export default function System() {
                         <Th>P-ID</Th>
                         <Th>Detection Start</Th>
                         <Th>Inference Speed</Th>
+                        <Th>CPU %</Th>
                       </Tr>
                     </Thead>
                     <Tbody>
                       <Tr>
                         <Td>{detectors[detector]['pid']}</Td>
                         <Td>{detectors[detector]['detection_start']}</Td>
-                        <Td>{detectors[detector]['inference_speed']}</Td>
+                        <Td>{detectors[detector]['inference_speed']} ms</Td>
+                        <Td>{cpu_usages[detectors[detector]['pid']]?.['cpu'] || '- '}%</Td>
```

Comment:
```suggestion
                        <Td>{cpu_usages[detectors[detector]['pid']]?.['cpu'] || '-'}%</Td>
```

Reply:
I actually copied this command (including that trailing space) from the Camera table below. Do you want it removed there to? I assumed it was like that for a reason

Reply:
Oops yeah I read that wrong.